Makefile

```diff
@@ -139,9 +139,13 @@ endif
 GIT_COMMAND ?= git
 ifeq ($(USE_COROUTINES), 1)
 	USE_FOLLY = 1
-	OPT += -DUSE_COROUTINES
+	OPT += -DUSE_COROUTINES -DHAVE_CXX11_ATOMIC
```

Comment:
Isn't HAVE_CXX11_ATOMIC unused / obsolete?

Reply:
Its used in glog/logging.h. I guess glog still supports very old compilers.